provisioning/miniprov/hare_mp/main.py

```diff
@@ -297,16 +298,20 @@ def disable_hare_consul_agent() -> None:
     execute(cmd)
 
 
+# Step 1
 def prepare(args):
     url = args.config[0]
-    utils = Utils(ConfStoreProvider(url))
+    motr_provider = get_motr_provider(url)
+    utils = Utils(ConfStoreProvider(url), motr_provider)
     stop_event = Event()
     conf_dir = get_config_dir(url)
     log_dir = get_log_dir(url)
     _create_consul_namespace(conf_dir)
     consul_starter = _start_consul(utils, stop_event, conf_dir, log_dir, url)
     utils.save_node_facts()
     utils.save_drives_info()
+    # import pudb; pudb.set_trace()
+    utils._save_metadrive_info(0, 0)
```

Comment:
Not needed. As we discussed, you can get the metadata directly within `config()` function (or it any other function that is supposed to generate CDF). So no persistence layer for metadata info is required.

Reply:
Addressed.